### PR TITLE
Add docstring for GenericAlias

### PR DESCRIPTION
**Pull request by emmatyping**
I wasn't sure exactly how to word this, but I took a crack at it.

### CODE REVIEW

**Comment by emmatyping** [comment]
The test failure looks unrelated.

File: Objects/genericaliasobject.c [hunk 1]
```diff
@@ -398,6 +398,8 @@ ga_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
 PyTypeObject Py_GenericAliasType = {
     PyVarObject_HEAD_INIT(&PyType_Type, 0)
     .tp_name = "types.GenericAlias",
+    .tp_doc = "A type to represent the type 'origin' as generic with respect to the type "
+              "arguments 'args'. See PEP 585 for more.",
```
Comment (gvanrossum) [inline Objects/genericaliasobject.c]: May I suggest a somewhat different text? How about:

"""
Represent a PEP 585 generic type

E.g. for t = list[int], t.origin is list and t.args is (int,).
"""

**Review by gvanrossum — COMMENTED** [review]
The test failure was fixed upstream, if you fetch and merge master it will go away. (I will try to merge master separately and just push it so the merge commit won't end up in your PR.)

**Comment by gvanrossum** [comment]
(Master is now merged into pep585.)

**Comment by gvanrossum** [comment]
I hope you don't mind that I'll just land my version so we can check this off. :-) Hope you are safe, be well.

**Comment by emmatyping** [comment]
Not a problem! I was working on some other things yesterday. I hope you are safe too :)

**Comment by gvanrossum** [comment]
You got some credit. :-)

**Comment by emmatyping** [comment]
That's very kind of you!

**Pull request merged**